lib/recurly/error.rb

```diff
@@ -0,0 +1,13 @@
+module Recurly
```

Comment:
I had to move ```error``` out into it's own file since it was defined in the file that was currently loading ```recurly.rb``` and after a ```require``` that depended on the class being defined. The options were to move this out to it's own file or to define the class before the require statement. IMHO doing the former was cleaner and more understandable. 

Paul